README.md

```diff
@@ -1,2 +1,457 @@
-# unione-php
-UniOne Email API PHP Library
+# UniOne PHP client
+
+[![Latest Stable Version](http://poser.pugx.org/unione/unione-php/v)](https://packagist.org/packages/unione/unione-php)
+[![Total Downloads](http://poser.pugx.org/unione/unione-php/downloads)](https://packagist.org/packages/unione/unione-php)
+
+This is the UniOne PHP SDK. This SDK contains methods for easily interacting
+with the UniOne API. For additional
+examples, please see our official documentation at https://docs.unione.io/en/web-api-ref#web-api
+
+# Installation
+
+To install the SDK, you will need to be using [Composer](http://getcomposer.org/)
+in your project.
+If you aren't using Composer yet, it's really simple! Here's how to install
+composer:
+
+```bash
+curl -sS https://getcomposer.org/installer | php
+```
+
+Use Composer to install the package:
+
+```bash
+composer require unione/unione-php
+```
+
+## Usage
+
+### Here's how to send a message using the SDK:
+More about the **UniOne** send mail API [https://docs.unione.io](https://docs.unione.io/en/web-api-ref#email).
+
+```php
+ $recipients = [
```

Comment:
It can be just a list of emails:

`['example@example.com', 'another@example.com'];`

Reply:
The mail array returns an error. It works like this

Reply:
Can you check `\Unione\Model\Email::setRecipients()` method? It's supposed to transform a list of emails into expected array.